### ISSUE TITLE
self-defence rights

### ISSUE THREAD

**Issue by KacperZajc** [liberland/constitution#526]
§XIV.1. All Persons shall have the right to use force, including disproportionate force so long as it is not apparently unreasonable, in defence against any Person, including any Agent of the Public Administration acting unlawfully or in error, posing a direct and real threat to (a) themselves, (b) their property, (c) constitutional Rights and/or (d) any other Person.

§XIV.6. All Persons shall have the right to use any force they deem fit against any Person who has been trespassing on the property they are in charge of and has refused to leave such property once requested.

**Comment by Auahi** [comment]
I would like the omission part to still be kept in the constitution. Also this text states that you can only defend someone who did not initiate the use of aggression. 

If I punch you and you start beating me ferociously, I think some text should be drafting allowing for others to protect me....

**Comment by KacperZajc** [comment]
There is nothing like that in this provision. 
What part are you referring to?

**Comment by KacperZajc** [comment]
fixed it

**Comment by Auahi** [comment]
§XIV.6. - I see problems arriving with this. Someone who wants to murder someone thats on their property can just kill them and state:

 _"They refused to leave"_

Because the person is no longer alive they can't testify that they didn't not refuse to leave.

**Comment by terrorist96** [comment]
I'm all for self-defense, but
> including disproportionate force so long as it is not apparently unreasonable,

sounds pretty scary here. It'll basically be impossible to convict murderers. You should only be allowed to use the same amount of force that you are reckoned with. Deadly force should only be ok where you fear for your life or great bodily harm.

I'd also change `real threat` to `true threat` to match US law.

And I also echo @Auahi's concern about killing trespassers. What if someone accidentally wanders into private property and is immediately shot? Or is told to leave but isn't given ample time to do so, or didn't understand the command to leave?

My suggestion:

> §XIV.1. All Persons shall have the right to use **reasonable** force~~, including disproportionate force so long as it is not apparently unreasonable,~~ in defence against any Person, including any Agent of the Public Administration acting unlawfully or in error, posing a direct and ~~real~~ **true** threat to (a) themselves, (b) their property, (c) **their** constitutional Rights and/or (d) any other Person.

§XIV.6. All Persons shall have the right to use ~~any~~ **reasonable** force ~~they deem fit~~ against any Person ~~who has been~~ trespassing on the property they are in charge of ~~and~~ **who** ~~has refused~~ **refuses** to leave such property once ~~requested~~ **commanded**.

**Comment by KacperZajc** [comment]
even in the UK now 'disproportionate' force is allowed in self-defence
Im fine with the American version
well it has always been my understanding that you should have the right to shoot anyone who is trespassing on your property, I would like to stick to that, however, I am open when it comes to alternative wording which would include better safeguard against people with itchy fingers

**Comment by terrorist96** [comment]
Yeah, you do have the right to shoot actual trespassers who mean harm to you/your family. Not someone who accidentally wandered onto your property. I feel like changing it to reasonable force versus any force they deem fit could lower the chances of someone shooting a trespasser instantly versus shooting them when they refuse to leave and continue to come towards you.

**Comment by liberlandcitizen** [comment]
*<<I feel like changing it to reasonable force versus any force they deem
fit could lower the chances of someone shooting a trespasser instantly
versus shooting them when they refuse to leave and continue to come towards
you.>>*

I second this

**Comment by KacperZajc** [comment]
Here is sth that combines all our positions:


§XIV.1. All Persons shall have the right to use reasonable force in defence against any Person, including any Agent of the Public Administration acting unlawfully or in error, posing a direct and true threat to (a) themselves, (b) their property, (c) their Constitutional Rights and/or (d) any other Person; should such a threat arise on the property a Person is in charge of, he or she shall have the right to use any force deemed fit. 

§XIV.6. All Persons shall have the right to use reasonable force to remove from the property they are in charge of any Person who is trespassing on that property and refuses to leave once commanded, regardless whether such a Person poses any threat.

self-defence in public - reasonable force
self-defence at home - unlimited force
removal of trespasser who is not dangerous from one's property - reasonable force

**Comment by terrorist96** [comment]
This is much better. But I see another thing we are forgetting. What if you have a guest on your property and they happen to be armed, and they have the ability to stop a trespasser? As written now, only the property owner/person in charge has the right to deter a trespasser.

**Comment by liberlandcitizen** [comment]
Could you guys explain the difference between unlimited force and
reasonable force? I tried looking this up but couldn't find unlimited
force. Am I right to believe this means that you can kill the person even
if they are not a direct and true threat to people or your property?

**Comment by terrorist96** [comment]
Unlimited force would be whatever the person wants, reasonable force would be an appropriate amount of force used depending on the threat level. So if someone slaps you, shooting and killing them would not be reasonable force.

**Comment by liberlandcitizen** [comment]
Could you kindly give an example or two of a time when somebody might use
unlimited force versus reasonable force?

And if I can borrow your example from above, if an unarmed person is on
your property and they slap you, unlimited force means the property owner
would have the right to kill the person in response?

**Comment by terrorist96** [comment]
Unlimited force would only be on property you own/are in charge of. Though I would argue that regardless of where you are, if you legitimately fear for your life, you have the right to kill the aggressor. I personally would remove the last clause from §XIV.1. because it's kinda superfluous.

**Comment by terrorist96** [comment]
> And if I can borrow your example from above, if an unarmed person is on
your property and they slap you, unlimited force means the property owner
would have the right to kill the person in response?

Yes, but unlimited force only applies if it satisfies the direct/true threat to (a)-(d). I would argue getting slapped in the face wouldn't constitute a true threat.

**Comment by liberlandcitizen** [comment]
<<Though I would argue that regardless of where you are, if you
legitimately fear for your life, you have the right to kill the aggressor.>>

Yes, I agree 100% but isn't that the exact definition of "reasonable"
force....you know....when force is actually reasonable? I'm still a little
uncertain under what circumstances "unlimited force" would be morally
justifiable and yet when "reasonable force" is not. What am I missing?
Examples would help.

**Comment by KacperZajc** [comment]
Im not sure you still get it - 

a person has a right to use reasonable force in self-defence in public space/on another's property
a person has a right to use any force he wants in self-defence on their own property
a person has a right to use reasonable force to remove any person WITHOUT A THREAT from their property 

So effectively we have 3 different right to self-defence, depending whether you are on your own property and whether a person you are using force against poses a threat.

**Comment by liberlandcitizen** [comment]
The only part I still don't get is what is the definition of unlimited
force? If I don't know the definition, there will be others who read this
constitution that don't know its meaning as well. And I'd still love an
example of when "unlimited force" would be justified when reasonable force
is not.

**Comment by KacperZajc** [comment]
reasonable force would be proportionate to the threat.
Unlimited might be disproportionate:

If sb punches you with his fist, proportionate force would be to punch him back with your fist, potentially with some object.
Disproportionate force would be to shoot him dead.

**Comment by liberlandcitizen** [comment]
​Thank you. Are we all in agreement that it's beneficial for a society to
allow unlimited (disproportionate) force? ​

**Comment by KacperZajc** [comment]
I think it is when it comes to a person standing on their own property.

**Comment by FellowTraveler** [comment]
"Force" is restraint. (Arrest.) For example, if I have my security guards grab you and drag you off my property, that is the use of force.

People have the right to use force against trespassers. A trespasser is someone who enters OR REMAINS on a property WITHOUT PERMISSION.

This means people do NOT have the right to shoot trespassers. However, if you inform the person that he is trespassing, then you have a right to use force to remove him from the property. And during that process you have a right to use deadly force in self-defense, if necessary. So he may end up getting shot in the end, but you don't have a right to just shoot him outright.

------------------------------------

"Deadly force" is the use--or threat to use--a degree of force that a reasonable person would say could cause death or serious bodily injury. Normally people do not have the right to use this kind of force except in defense of life/liberty/property.

For example, if I am in public and someone assaults me, or if I see a robbery in progress, or a home invader has just broken a window to enter my house, etc. In those cases I definitely have a right to use deadly force.

(Otherwise I would not be able to delegate this authority to my public servants, and thus the police would also have no right to use deadly force in those circumstances.)

Basically the people have the exact same authority to use deadly force as the police do. Otherwise, it would be impossible for the people to delegate this authority to the police in the first place, and thus the police would have no legitimate power.

------------------------------------

I am very disturbed by the wording "A person shall have the right to use force in defense..."

The wording should rather be, "Government shall not infringe upon a person's natural right to use force in defense..."

See the difference? It's an important one.

**Comment by ConnectorLL** [comment]
It's important question to be discussed. It's literally about life and death.
So let's see what we want to achieve. 
-protection of proprety
-ability to self defence
What we don't want to achieve:
-a lot of dead people

I think the force shouldn't be overused, so deathly force could and should be only used when your life or life of someone else is under direct threat. I would not like to see people dead just becouse they stepped on someones proprety or even when they refuse to move out, if they don't present thread of life. In those situations physical force should be allowed to move them out of proprety and only then if they would like to attack, you could use deathly force. Even If you use weapon to scare intruder off your proprety you should not kill him doing that.  

Let's see an example: someone comes by on a party. They get drunk and become unwanted. They are still on someone elses proprety. Could the owner just shoot and kill him? Could the owner shoot him and kill him when he refuses to go?

I think life is sacred thing and giving right to someone to take it away shouldn't be taken lightly.

I think we also don't want to see excessive use of weapons in police like it is in USA.

Check different models that are used throughout the world about this topic, select one that are doing well and we would like to have.

**Comment by liberlandcitizen** [comment]
I agree with everything said in the previous message, most of all that
"deathly force could and should be only used when your life or life of
someone else is under direct threat."

**Comment by KacperZajc** [comment]
exactly what part of my proposal do you not agree with?

**Comment by KacperZajc** [comment]
I dont see any opposition so I am closing this discussion.

**Comment by liberlandcitizen** [comment]
There was plenty of opposition in this thread. Unlimited force seems to
allow people to kill beyond what is reasonable/necessary for protection. I
admit I am not legally trained but others seem to share my concerns.
